examples/discord_bot.py

```diff
@@ -140,7 +140,7 @@ async def current_war_status(ctx, clan_tag):
         await ctx.send("You didn't give me a proper tag!")
         return
 
-    e = discord.Embed(colour=discord.Colour.blue())
+    e = discord.Embed(colour=discord.Color.blue())
```

Comment:
This should be left as `Colour`. [`Color` is an alias for `Colour`](https://discordpy.readthedocs.io/en/latest/api.html#discord.Colour)